main.js

```diff
@@ -0,0 +1,29 @@
+var $myButton = document.querySelector('button')
+var $count = document.querySelector('h3')
+var countTime = false
+var intervalID;
+
+function changeDigit() {
+  var elapsedTime = parseInt($count.textContent, 10)
+  if (countTime === true) {
+    elapsedTime += 1 //
+    $count.textContent = elapsedTime
+  }
+}
+
+function changeInterval() {
+  countTime = !countTime //toggles between on and off through boolean
```

Comment:
Please remove all comments. It's ok to include them before you open a pull request, but delete them before opening one. Assume your audience and read JavaScript. Use meaningful variable and function names instead of adding comments.

Reply:
got it, thanks!